File: src/main/java/dev/sky_lock/pocketlifevehicle/task/SubmergedWarning.java
package dev.sky_lock.pocketlifevehicle.task;

import org.bukkit.ChatColor;
import org.bukkit.entity.Player;

/**
 * @author sky_lock
 */

class SubmergedWarning extends AbstractWarning {

    void send(Player player) {
        super.send(player, ChatColor.GOLD + "乗り物は" + getCount() + "秒後に水没します");
    }
}

File: src/main/kotlin/dev/sky_lock/pocketlifevehicle/task/SubmergedWarning.kt
package dev.sky_lock.pocketlifevehicle.task

import dev.sky_lock.pocketlifevehicle.extension.chat.Line
import org.bukkit.entity.Player

/**
 * @author sky_lock
 */
class SubmergedWarning : AbstractWarning() {
    fun send(player: Player) {
        super.send(player, Line().gold("乗り物は" + super.count + "秒後に水没します"))
    }
}